### DISCUSSION TITLE
Amazing plugin...

### DISCUSSION THREAD

**Discussion by johnsonblue75** [snezhig/obsidian-front-matter-title#94] [General]
Wow. I have to take my son to school but when I get back I'm buying you thermoses of coffee, maybe even a coffee plant (tree)? This plugin is amazingly helpful. Thank you. Wonderful work and it makes things so much more intuitive.

**Comment by snezhig** [comment]
Thank you for the feedback. I will try to do more things that will make your user experience with the plugin more enjoyable.

**Reply by johnsonblue75** [reply]
You're welcome. And I just sent your coffee! Heh.

Okay, love, love, LOVE this plugin. I will tell you about an odd "challenge" I sometimes have. When I'm USING Osidian it's not a problem, but when I'm playing with/customizing obsidian it's a little harder (but rare circumstances) because I happen to have VScode open - I prefer editing in there - so I have it open and they're on separate monitors.

The first time it took me a little bit because I was looking at the file and using it in obsidian, then I turned to Vscode and couldn't find. Maybe ten minutes later I realized this was happening, lol.

Anyway, you can't really fix that, but I would encourage you to put that somewhere so people are reminded of it - and also - perhaps to put a YAML for the the actual filename or Obsidian path - that's what I'm going to do. 

Thanks! Ooh, maybe you could toggle or have a shortcut within obsidian to show underlying file name that might be quite helpful.

Great work.

**Reply by snezhig** [reply]
Thank you for the coffee.
I have added some new commands in my current working branch.
One command lets you disable features, the another one lets you reload\enable them. So you can bind hot keys for this commands and toggle plugin to see real filenames.
It's easiest and fastest things I can do right now.
Also I have another thoughts that I can add in the future, but they needs more time to be realized.
